content/pages/sprint/2023/05-hsinchu.rst

```diff
@@ -0,0 +1,82 @@
+========================================
+Scisprint 2023 May in Hsinchu
+========================================
+
+:date: 2023-03-17 17:57
```

Comment:
This date seems strange.

Reply:
I have just updated the date to the correct one. Can you please verify it for me?

Reply:
I have made another PR #199 .